cli/tests/integrations/test_package.py

```diff
@@ -18,10 +18,15 @@
 
 
 def setup_module(module):
+    # add universe-server with static packages
+    assert_command(
+        ['dcos', 'marathon', 'app', 'add', 'tests/data/universe-v3-stub.json'])
+
     assert_command(
         ['dcos', 'package', 'repo', 'remove', 'Universe'])
-    repo = "https://github.com/mesosphere/universe/archive/cli-test-4.zip"
-    assert_command(['dcos', 'package', 'repo', 'add', 'test4', repo])
+
+    repo = "http://universe.marathon.mesos:8085/repo"
```

Comment:
This url should never change, only the the docker container referenced in `tests/data/universe-v3-stub.json`